### DISCUSSION TITLE
Non-docker instilation

### DISCUSSION THREAD

**Discussion by HelpfulFix** [bluenviron/mediamtx#1419] [General]
The instructions don't go into much detail about the non-docker installation.  

I tried a "make binaries" but the make file is looking for docker which isn't installed on this system.

**Comment by aler9** [comment]
Hello,

If you want to perform a standard installation, prebuilt binaries are available in the [release page](https://github.com/aler9/rtsp-simple-server/releases).

If you want to compile from source, instructions are available in the [README](https://github.com/aler9/rtsp-simple-server#compile-and-run-from-source); the command is `go run .`, not `make binaries`.

[a comment by an automated account is not shown]
